flybywiresim/aircraft
# fix(texture): lessen emissive integ texture pink shift

Fixes #[issue_no]

## Summary of Changes

Credit to FinalLight.

Modifies the emissive textures to be less pink shifted and more amber again.

## Screenshots (if necessary)



After:

![image](https://github.com/flybywiresim/aircraft/assets/15316958/da6d9896-4bff-4f06-b7e7-067419ac6968)
![image](https://github.com/flybywiresim/aircraft/assets/15316958/708507a5-e2d5-4483-810e-534e9c410b29)
![image](https://github.com/flybywiresim/aircraft/assets/15316958/8d336eb2-e5fe-4883-8c3f-e45172d5d835)

## References





## Additional context



Discord username (if different from GitHub): 2Cas

## Testing instructions

1. Load in, turn integrated lighting up, observe that it is different from master (see screenshots before/after)
2. Ensure there are no texture errors (pink/black cubes) in the cockpit.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
